Composer/packages/client/src/components/AddRemoteSkillModal/BrowserModal.tsx

```diff
@@ -36,7 +37,14 @@ export const BrowserModal = (props) => {
         text={formatMessage('Browse')}
         onClick={onClickOpen}
       />
-      <input ref={inputFileRef} accept=".zip,.rar,.7zip" style={{ display: 'none' }} type="file" onChange={onChange} />
+      <input
+        ref={inputFileRef}
+        accept=".zip,.rar,.7zip"
+        max-file-size="1048576"
```

Comment:
how is this used? 
you should manually check the uploaded file size in the change callback and make sure it's within the limits of 1 GB use the following in your calculations instead of 1048576 ```const fileSizeLimit = 1024 * 1024```